eclair-core/src/main/scala/fr/acinq/eclair/channel/publish/ReplaceableTxFunder.scala

```diff
@@ -312,13 +315,13 @@ private class ReplaceableTxFunder(nodeParams: NodeParams,
     }
   }
 
-  private def sign(fundedTx: ReplaceableTxWithWitnessData, txFeerate: FeeratePerKw, amountIn: Satoshi): Behavior[Command] = {
+  private def sign(fundedTx: ReplaceableTxWithWitnessData, txFeerate: FeeratePerKw, amountIn: Satoshi, packageWeight: Int): Behavior[Command] = {
```

Comment:
I don't think you need this `packageWeight` argument, and even if you did this wouldn't work because the weight you're passing in doesn't account for the witnesses. I believe you can simply use the weight of the signed tx once you have it, which reduces the number of changes to this file.

Reply:
We need to re-compute the weight that was used to fund the transaction (which may include the weight of the tx it wants to bump, using the weight of the signed tx is not enough) but it can be recomputed locally instead of being passed around, see https://github.com/ACINQ/eclair/pull/2613/commits/f9fa8a7da11605335a0610c0dd61b3dcfca36fa5